src/pages/HistoricalStatistics/Difficulty/index.tsx

```diff
@@ -0,0 +1,63 @@
+// react
+import { useEffect, useState } from 'react';
+// third party
+import { Skeleton } from '@material-ui/lab';
+// application
+import { Container } from '@pages/HistoricalStatistics/StatisticsOvertime.styles';
+import * as URLS from '@utils/constants/urls';
+import { useFetch } from '@utils/helpers/useFetch/useFetch';
+import { PeriodTypes, transformDifficultyInfo } from '@utils/helpers/statisticsLib';
+import { CHART_DEFAULT_PERIOD, periods, info } from '@utils/constants/statistics';
+import { IStatistic, TLineChartData } from '@utils/types/IStatistics';
+import { useBackgroundChart } from '@utils/hooks';
+import { EChartsLineChart } from '../Chart/EChartsLineChart';
+
+function Difficulty() {
+  const [chartData, setChartData] = useState<TLineChartData | null>(null);
+  const [currentBgColor, handleBgColorChange] = useBackgroundChart();
+  const [period, setPeriod] = useState<PeriodTypes>(CHART_DEFAULT_PERIOD);
+  const fetchStats = useFetch<{ data: Array<IStatistic> }>({
+    method: 'get',
+    url: URLS.GET_STATISTICS,
+  });
```

Comment:
I can see no loading and no error handling here,
I suggest to use `react-query`: it takes care of caching, provide loading and error state.